api/model/model_types.go

```diff
@@ -0,0 +1,139 @@
+//
+// Copyright (c) 2012-2018 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package model
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/docker/docker/api/types"
+	"github.com/eclipse/che-lib/websocket"
```

Comment:
you can reference github.com/gorilla/websocket  directly here.
We have version 361d4c0ffd78338ebe0a9e6320cdbe115d7dc026 approved in https://dev.eclipse.org/ipzilla/show_bug.cgi?id=10452

Reply:
Oh, yes. Thanks, it's a good idea.